debian-config-jobs

```diff
@@ -1527,8 +1527,82 @@ function jobs ()
 	;;
 
 
+	
+	"Dtc" )
+		# Check if dtc command installed and install it if missing
+		if ! command -v dtc 2> /dev/null
+		then
+			echo "device tree compiler (dtc) could not be found, ask for installing it"
+			sudo apt install dtc
+			if ! command -v dtc 2> /dev/null
+			then
+				echo "Failed to install device tree compiler (dtc), exiting!"
+				exit
+			fi
+		else
+			echo "dtc already installed!"
+		fi
+
+		# Read current used dtb in /boot/armbianEnv.txt
```

Comment:
Rarely its defined in armbianEnv.txt

Reply:
Ok, after investigating device tree and information about current hardware, I hope my last commit is a better solution (tested on 3 different boards with AW, RK and AML socs).

Reply:
Let's test that ... another approach would be to dump a DT

(also not sure if this is working everywhere, but we could start to ignore legacy kernels regarding this)
dtc -I fs -O dts /sys/firmware/devicetree/base

... and use "custom.dtb" for this?